### ISSUE TITLE
Texture Bug Hotbar

### ISSUE THREAD

**Issue by PocketMiner92** [PowerNukkit/PowerNukkit#1318]
# 🐞 I found a bug
 

### 📸 Screenshots / Videos

![IMG_20220211_131723168.jpg](https://user-images.githubusercontent.com/96331694/153591099-18dc9d85-0161-4e35-bd38-382f0da5ad74.jpg)

### ▶ Steps to Reproduce

1. Run command '...'
3. Put '....' at '...'

### 📋 Debug information

* PowerNukkit version: git-37b1ff9
* Debug link: ✍
* Timings link (if relevant): ✍

**Comment by irrld** [comment]
I can confirm this happens when slab is placed to upper side of a block. It changes the item state instead of cloning it before placing.

**Comment by irrld** [comment]
ItemBlock#getBlock seems to not clone the block when called, that might be the issue.

**Comment by irrld** [comment]
This issue doesn't happen to the dropped item, it happens when placing so the title isn't quite correct.

**Comment by alikindsys** [comment]
yeah, i had to guess on what the issue was from "Texture Bug Hotbar" and a screenshot, thanks for helping out.

**Comment by alikindsys** [comment]
Bug has been replicated on c9f67c981c4f3c3a8957bb90edc6e21102aa912a.

https://user-images.githubusercontent.com/36565196/177085676-3d330dfc-d4f6-482d-aca5-8c35e70dc4e2.mp4

**Comment by alikindsys** [comment]
This is a client-server desync issue. If you reconnect your items are fine, its a visual glitch. 
Why is powernukkit sending the inverted slab needs to be investigated, but that's all I got for now.  
Since the items don't persist after a relog I'll reduce the priority.

This bug **only** happens in *survival* mode.

**Comment by GoodLucky777** [comment]
Duplicate of #977
